scrapyd_k8s/api.py

```diff
@@ -148,5 +148,11 @@ def run():
     if config_username is not None and config_password is not None:
         enable_authentication(app, config_username, config_password)
 
+    if config.joblogs() is not None:
```

Comment:
Hey Wesley, thanks for your review. I don't have anything against property or decorator pattern, but since we don't use it, it really belongs to another PR about refactoring of the whole project, not my logging feature.